docs/dev-guide/integration/rest-api/server_plugins.rst

```diff
@@ -0,0 +1,336 @@
+Server's Plugins API
```

Comment:
Maybe just "Plugins API". I think you are trying to be clear that this is not the API that the plugins themselves use with pulp, but since all the other API's are on the server also, that might introduce some confusion.